README.md

```diff
@@ -93,6 +93,7 @@ Below the above include the following:
 ```
 - name: Print the Cloud Formation Linter Version & run Linter.
   run: |
+    setopt -s globstar # enable globbing
     cfn-lint --version
     cfn-lint -t ./**/*.yaml
```

Comment:
```suggestion
    cfn-lint --version
    cfn-lint -t ./template.yml
```
Alternatively we could just change this to a single file.

Reply:
I'd be partial to calling out the possible need for globbing, or at least pointing to where it's mentioned in the core repo (see the first "Note" beneath [here](https://github.com/aws-cloudformation/cfn-lint#basic-usage)) in the text around this.

But I've only ever interacted with your all's tool once, so idk how it tends to get used.